cached_network_image/lib/src/image_provider/cached_network_image_provider.dart

```diff
@@ -49,7 +44,7 @@ class CachedNetworkImageProvider
   final double scale;
 
   /// Listener to be called when images fails to load.
-  final image_provider.ErrorListener? errorListener;
+  final ValueChanged<Object>? errorListener;
```

Comment:
Why did you change this to ValueChanged? Sounds a bit weird for a callback as it's not really changing any value, but it's just returning an error.

Reply:
ValueChanges<Object> it is default flutter typedef.
void Function(Object)

Reply:
I know, but still don't think it's an appropriate use: https://api.flutter.dev/flutter/foundation/ValueChanged.html

> Signature for callbacks that report that an underlying value has changed.

Imo it's misusing a typedef just because it works, not because it's the same intent.